ui/src/containers/BuildDetails.svelte

```diff
@@ -95,6 +95,9 @@
           <div class="text-center">
             <span class="text-xl font-sans block lg:inline-block text-gray-600">Last scanned: {formatDistanceToNow(new Date(data.summary.buildDate), {addSuffix: true})} at {format(new Date(data.summary.buildDate), 'hh:mm')}</span>
```

Comment:
looks like all these code are duplicated between different tabs. we need to move it to common reusable component